src/main/resources/db/migration/v0.31.0.1__linkml_v2.1.0_update.sql

```diff
@@ -0,0 +1,1173 @@
+ALTER TABLE doterm ADD CONSTRAINT doterm_pkey PRIMARY KEY (id);
+ALTER TABLE dpoterm ADD CONSTRAINT dpoterm_pkey PRIMARY KEY (id);
+ALTER TABLE ecoterm ADD CONSTRAINT ecoterm_pkey PRIMARY KEY (id);
+ALTER TABLE emapaterm ADD CONSTRAINT emapaterm_pkey PRIMARY KEY (id);
+ALTER TABLE experimentalconditionontologyterm ADD CONSTRAINT experimentalconditionontologyterm_pkey PRIMARY KEY (id);
+ALTER TABLE fbdvterm ADD CONSTRAINT fbdvterm_pkey PRIMARY KEY (id);
+ALTER TABLE gene ADD CONSTRAINT gene_pkey PRIMARY KEY (id);
+ALTER TABLE genomicentity ADD CONSTRAINT genomicentity_pkey PRIMARY KEY (id);
+ALTER TABLE goterm ADD CONSTRAINT goterm_pkey PRIMARY KEY (id);
+ALTER TABLE hpterm ADD CONSTRAINT hpterm_pkey PRIMARY KEY (id);
+ALTER TABLE materm ADD CONSTRAINT materm_pkey PRIMARY KEY (id);
+ALTER TABLE miterm ADD CONSTRAINT miterm_pkey PRIMARY KEY (id);
+ALTER TABLE mmoterm ADD CONSTRAINT mmoterm_pkey PRIMARY KEY (id);
+ALTER TABLE mmusdvterm ADD CONSTRAINT mmusdvterm_pkey PRIMARY KEY (id);
+ALTER TABLE modterm ADD CONSTRAINT modterm_pkey PRIMARY KEY (id);
+ALTER TABLE molecule ADD CONSTRAINT molecule_pkey PRIMARY KEY (id);
+ALTER TABLE mpathterm ADD CONSTRAINT mpathterm_pkey PRIMARY KEY (id);
+ALTER TABLE mpterm ADD CONSTRAINT mpterm_pkey PRIMARY KEY (id);
+ALTER TABLE ncbitaxonterm ADD CONSTRAINT ncbitaxonterm_pkey PRIMARY KEY (id);
+ALTER TABLE obiterm ADD CONSTRAINT obiterm_pkey PRIMARY KEY (id);
+ALTER TABLE ontologyterm_isa_parent_children ADD CONSTRAINT ontologyterm_isa_parent_children_pkey PRIMARY KEY (isachildren_id, isaparents_id);
+ALTER TABLE ontologyterm_isa_ancestor_descendant ADD CONSTRAINT ontologyterm_isa_ancestor_descendant_pkey PRIMARY KEY (isadescendants_id, isaancestors_id);
+ALTER TABLE patoterm ADD CONSTRAINT patoterm_pkey PRIMARY KEY (id);
+ALTER TABLE phenotypeterm ADD CONSTRAINT phenotypeterm_pkey PRIMARY KEY (id);
+ALTER TABLE pwterm ADD CONSTRAINT pwterm_pkey PRIMARY KEY (id);
+ALTER TABLE reference ADD CONSTRAINT reference_pkey PRIMARY KEY (id);
+ALTER TABLE roterm ADD CONSTRAINT roterm_pkey PRIMARY KEY (id);
+ALTER TABLE rsterm ADD CONSTRAINT rsterm_pkey PRIMARY KEY (id);
+ALTER TABLE soterm ADD CONSTRAINT soterm_pkey PRIMARY KEY (id);
+ALTER TABLE stageterm ADD CONSTRAINT stageterm_pkey PRIMARY KEY (id);
+ALTER TABLE uberonterm ADD CONSTRAINT uberonterm_pkey PRIMARY KEY (id);
+ALTER TABLE variant ADD CONSTRAINT variant_pkey PRIMARY KEY (id);
+ALTER TABLE vtterm ADD CONSTRAINT vtterm_pkey PRIMARY KEY (id);
+ALTER TABLE wbbtterm ADD CONSTRAINT wbbtterm_pkey PRIMARY KEY (id);
+ALTER TABLE wblsterm ADD CONSTRAINT wblsterm_pkey PRIMARY KEY (id);
+ALTER TABLE wbphenotypeterm ADD CONSTRAINT wbphenotypeterm_pkey PRIMARY KEY (id);
+ALTER TABLE xbaterm ADD CONSTRAINT xbaterm_pkey PRIMARY KEY (id);
+ALTER TABLE xbedterm ADD CONSTRAINT xbedterm_pkey PRIMARY KEY (id);
+ALTER TABLE xbsterm ADD CONSTRAINT xbsterm_pkey PRIMARY KEY (id);
+ALTER TABLE xcoterm ADD CONSTRAINT xcoterm_pkey PRIMARY KEY (id);
```

Comment:
Wondering if these should be "pk" vs "pkey"

Reply:
"pkey" is the default suffix that's used on all our primary keys where we don't specify a name in the table definition

Reply:
Oh ok... I thought it was "pk" as the others are "uk" and "fk" but whatever.